apis/kubedb/v1alpha2/types.go

```diff
@@ -139,3 +139,28 @@ type CoordinatorSpec struct {
 	// +optional
 	SecurityContext *core.SecurityContext `json:"securityContext,omitempty"`
 }
+
+// HealthCheckSpec defines attributes of the health check
+type HealthCheckSpec struct {
+	// How often (in seconds) to perform the health check.
+	// Default to 10 seconds. Minimum value is 1.
+	// +optional
+	// +kubebuilder:default:=10
+	PeriodSeconds *int32 `json:"periodSeconds,omitempty"`
```

Comment:
`Frequency` would be better I think

Reply:
We've chosen the terms from Kubernetes probe (liveness/readiness). https://github.com/kubernetes/api/blob/master/core/v1/types.go#L2190-L2226